draft-fossati-tls-attestation.md

```diff
@@ -1061,6 +1075,24 @@ the TLS server will terminate the exchange.
 
 TBD.
 
+# Privacy Considerations {#priv-cons}
+
+In this section, we are assuming that the Attester is a TLS client, representing an individual person.
+We are concerned about the potential leakage of unnecessary information about that person, such as the correlation of different connections initiated by them.
```

Comment:
```suggestion
We are concerned about the potential leakage of privacy sensitive information about that person, such as the correlation of different connections initiated by them.
```

Reply:
cool, done in [c749575](https://github.com/tls-attestation/draft-tls-attestation/pull/59/commits/c7495751974e1b765fbc4f3a2aae832d42a27744)